pep-0599.rst

```diff
@@ -164,7 +164,7 @@ the ``manylinux2014`` tag:
    they may only depend on the following maximum versions::
 
        GLIBC_2.17
-       CXXABI_1.3.7
+       CXXABI_1.3.7, CXXABI_TM_1 is also allowed
```

Comment:
```suggestion
       CXXABI_1.3.7 (CXXABI_TM_1 is also allowed)
```
Might be good to reference https://gcc.gnu.org/ml/libstdc++/2019-10/msg00023.html here as well.